### PR TITLE
Added Vbat Sag Compensation to UI

### PR DESCRIPTION
**Pull request by Asizon**
Added Vbat Sag Compensation to UI, when its enabled the sag value goes to default standard of 100 else it goes to 0(disabled).

image

### CODE REVIEW

File: src/js/tabs/pid_tuning.js [hunk 1]
```diff
@@ -451,8 +451,20 @@ TABS.pid_tuning.initialize = function (callback) {
             });
             ffInterpolateCheck.change();
 
+            // Vbat Sag Compensation
+            const vbatSagCompensationCheck = $('input[id="vbatSagCompensation"]');
+
+            vbatSagCompensationCheck.prop('checked', ADVANCED_TUNING.vbat_sag_compensation !== 0);
+            $('input[name="vbatSagValue"]').val(ADVANCED_TUNING.vbat_sag_compensation > 0 ? ADVANCED_TUNING.vbat_sag_compensation : 100);
```
Comment (McGiverGim) [inline src/js/tabs/pid_tuning.js:458]: When it is zero the value is 100? Maybe this is the default value or something similar?

File: src/js/tabs/pid_tuning.js [hunk 2]
```diff
@@ -451,8 +451,20 @@ TABS.pid_tuning.initialize = function (callback) {
...
+
+            vbatSagCompensationCheck.change(function() {
+                const checked = $(this).is(':checked');
+                $('.vbatSagCompensation .suboption').toggle(checked);
+            });
+            vbatSagCompensationCheck.change();
```
Comment (McGiverGim) [inline src/js/tabs/pid_tuning.js:464]: You can add the change action directly to the change function, something like:
```
vbatSagCompensationCheck.change(function() {
                const checked = $(this).is(':checked');
                $('.vbatSagCompensation .suboption').toggle(checked);
            }).change();

File: src/js/tabs/pid_tuning.js (hunk 1 above)
Comment (Asizon) [reply src/js/tabs/pid_tuning.js:458]: @McGiverGim yes, 100 is our default value when is enabled this feature recomended by @ctzsnooze

File: src/js/tabs/pid_tuning.js (hunk 2 above)
Comment (Asizon) [reply src/js/tabs/pid_tuning.js:464]: okey, I dont know why I not included directly in the function since I made FF Interpolate, making changes now

**Comment by Asizon** [comment]
Making some changes in the tooltip, please dont merge yet

**Comment by Asizon** [comment]
Done, updated tooltip as @ctzsnooze requested and added % after number box

image

**Pull request merged**